docs/faq.md

```diff
@@ -0,0 +1,33 @@
+# General FAQ
+
+## AMD GPUs crashing
+
+Changing resolution, using DRI_PRIME and doing various other things can cause crashes, but running the command `echo high| sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level` or adding the `amdgpu.dpm=0` to the kernel commandline stops these crashes.
```

Comment:
There are less severe gpu crashes that can occur, these steps don't help with those so probably clarify that the *whole system* dies with the same symptoms as a t2 panic and it'll say there was a cpu caterr